scripts/vagrant/install-provenance.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
I'd suggest splitting these into `build-provenance.sh` and `install-provenance.sh`?

Reply:
If anything, I might delete this script. It doesn't quite work, especially the part around `postgresql-setup-conf.sql`.

Reply:
That sounds like a way to reduce the confusion.